clients/java/revapi.json

```diff
@@ -91,6 +91,19 @@
           "code": "java.class.removed",
           "old": "class io.camunda.zeebe.client.protocol.rest.UserTaskUpdateRequestChangeset",
           "justification": "The classes/methods are only internal and are not user-facing, and as such does not need to maintain any backwards compatibility guarantees."
+        },
+        {
+          "ignore": true,
+          "code": "java.method.returnTypeTypeParametersChanged",
+          "old": "method java.util.List<io.camunda.zeebe.client.api.response.Process> io.camunda.zeebe.client.api.response.DeploymentEvent::getProcesses()",
+          "new": "method java.util.List<io.camunda.zeebe.client.api.response.ProcessDefinition> io.camunda.zeebe.client.api.response.DeploymentEvent::getProcesses()",
+          "justification": "The classes/methods are only internal and are not user-facing, and as such does not need to maintain any backwards compatibility guarantees."
```

Comment:
💭 Unfortunately, this is not true. This IS the result of a client command and definitely user-facing. This breaks user space. Anyone using the result of a "deploy resources" client command will have to adjust their code with this change.